app/src/components/ArtifactTimeline.tsx

```diff
@@ -0,0 +1,102 @@
+import { Center, ScrollArea, Space, Text, Timeline } from '@mantine/core';
+import { Message } from 'tabler-icons-react';
+import { useEffect, useState } from 'react';
+import ArtifactComments from './ArtifactComments';
+import { ArtifactCommentProps } from './ArtifactComments';
+
+interface ArtifactTimelineProps {
+  extensions?: fhir4.Extension[];
+}
+
+type commentProps = {
+  author?: string;
+  date?: string;
+  body?: string;
+  type?: string;
+};
+
+function addArtifactComment({ date, body, author }: ArtifactCommentProps) {
+  return (
+    <div>
+      <ArtifactComments date={date} body={body} author={author} />
+    </div>
+  );
+}
+
+function noCommentsAvailable() {
+  return (
+    <div>
+      <Center>
+        <Text color="red">
+          <i>No Comments Found for Artifact</i>{' '}
+        </Text>
+      </Center>
+    </div>
+  );
+}
+
+export default function ArtifactTimeline(extensionArr: ArtifactTimelineProps) {
```

Comment:
```suggestion
export default function ArtifactTimeline({ extensions }: ArtifactTimelineProps) {
```
You can just get `extensions` directory so that you don't have to do `extensionArr?.extensions` below!

Reply:
Great catch!